config/routes.rb

```diff
@@ -10,6 +10,8 @@
   get '/422', to: 'errors#unprocessable_entity', via: :all
   get '/500', to: 'errors#internal_server_error', via: :all
 
+  get '/email-preferences', to: 'pages#email_preferences'
```

Comment:
This confuses me. I wanted to see it here https://ey-recovery-pr-659.london.cloudapps.digital/email-preferences but the review app gives a 500. I am asking because I have just seeded `static` in the CMS `master` env.

Reply:
So do you need this? You can test against the CMS using `master` now.

Reply:
This is the initial redirect path for returning users who have not given their email preferences - should I define this route elsewhere?